### ISSUE TITLE
Обязательность наличия класса в каждом узле

### ISSUE THREAD

**Issue by oficrejosef** [bem-site/bem-forum-content-ru#1340]
Доброе утро. Подскажите, пожалуйста, в случае использования БЭМ в качестве методологии именно для именования классов в своем проекте(без использования bemjs и прочего) является ли обязательным наличие класса у элемента блока, если у данного узла попросту нет никаких стилей? То есть что-то вроде
```html
<div class="block">
<div>
<span class="block__elem"></span>
<div class="block__elem"></div>
</div>
</div>
```

**Comment by tadatuta** [comment]
@oficrejosef Нет, не является. БЭМ — не догма, а способ упростить разработку. Так что нет причин делать что-либо, что усложняет жизнь и противоречит здравому смыслу.
